packages/matchbox/src/components/Tabs/Tab.tsx

```diff
@@ -1,35 +1,36 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import { UnstyledLink } from '../UnstyledLink';
+import { UnstyledLink, UnstyledLinkProps } from '../UnstyledLink';
 import { tabStyles } from './styles';
-import { deprecate } from '../../helpers/propTypes';
+import * as Polymorphic from '../../helpers/types';
+
+export type TabProps = UnstyledLinkProps & {
+  selected?: number;
+  fitted?: boolean;
+  content?: React.ReactNode;
+  index?: number;
+  onClick?: (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>, index: number) => void;
```

Comment:
I don't think we need the second `MouseEvent` parameter here but it's not hurting anything so your call